locale/en/Pacifist.cfg

```diff
@@ -1,3 +1,18 @@
+[gui]
+guns=
+weapons=
+ammo=
+
+[inventory-full-message]
+guns=
+ammo=
+
+[inventory-restriction]
+cant-be-used-as-ammo-for=
```

Comment:
I could not reproduce these messages, where are they used?

Reply:
If you have an item on your cursor and click on the slot, then those show as floating text. I'll add them back in reworded to refer to tools.